### DISCUSSION TITLE
MixMonitor

### DISCUSSION THREAD

**Discussion by emassen** [mikopbx/Core#306] [Q&A]
Hy,

* Party A places a call to Party B
* Party A places an attended transfer to Party C
Call recording works up to this point – the recording of the conversation
between Party A and Party B, and the portion of the conversation between Party A
and Party C is recorded
* Party A now hangs up
* Call recording is now stopped
* Party B and Party C are now speaking (unrecorded)

**Comment by boffart** [comment]
specify the PBX version

**Reply by emassen** [reply]
Hi,

2022.1.42-dev

Thank You.

**Comment by boffart** [comment]
Run the command
`sqlite3 /storage/usbdisk1/mikopbx/astlogs/asterisk/cdr.db 'select * from cdr';`
attach the output here.

If the table is not empty, then try to clear it
`sqlite3 /storage/usbdisk1/mikopbx/astlogs/asterisk/cdr.db 'delete from cdr_general';`

Try again:
> Party A places a call to Party B
> Party A places an attended transfer to Party C
> Call recording works up to this point – the recording of the conversation
> between Party A and Party B, and the portion of the conversation between Party A
> and Party C is recorded
> Party A now hangs up
> Call recording is now stopped
> Party B and Party C are now speaking (unrecorded)
